sim/mcuboot-sys/build.rs

```diff
@@ -167,11 +169,41 @@ fn main() {
         }
     }
 
+    if enc_ec256 {
```

Comment:
This conflicts with #571, however it's fine to leave it, since that is still marked as WIP, and this is close to ready.